### ISSUE TITLE
Add SDK methods for starting the Kurtosis Engine/Gateway

### ISSUE THREAD

**Issue by bsamuels453** [kurtosis-tech/kurtosis#1722]
### Background & motivation

The tool I'm working(Attacknet) on is intended to encapsulate multiple tools and orchestrate them with no user input.

Right now, there's no way for Attacknet to start the Kurtosis engine or the Kurtosis gateway using the SDK. Instead, we have to prompt the user to manually run these commands on their machine before starting attacknet. 

### Desired behaviour

It would be nice if there were SDK methods to start the engine/gateway. This might include logic to determine whether there's already a running engine/gateway so we don't start two engines on accident.

### How important is this to you?

Nice to have; this feature would make using Kurtosis more enjoyable.

### What area of the product does this pertain to?

SDK: the Software Development Kit libraries - Typescript, Go, etc.
